frontend/src/app/features/about-page/about-page.component.scss

```
@@ -0,0 +1,163 @@
+  content: "";
+  position: absolute;
+  width: 600px;
+  height: 600px;
+  background: radial-gradient(circle, rgba(107,78,49,0.35), transparent 70%);
+  top: -200px;
+  right: -200px;
+  filter: blur(80px);
+  animation: floatGlow 12s ease-in-out infinite alternate;
+}
+
+@keyframes floatGlow {
```

Stylelint (17.2.0)

[error] Expected keyframe name "floatGlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)